todo_app/data/Trello.py

```diff
@@ -118,3 +125,37 @@ def update_to_done(cls, item_id):
         list_id = trello.get_done_list_id()
         response = trello.move_item(item_id, list_id)
         return response
+
+    @staticmethod
+    def create_board(board_name):
+        params = trello.headers | {ResponseFilter.name: board_name}
+        print(params)
```

Comment:
Let's remove this; this prints out our api token & key to the log!

Reply:
All the Review comments are incorporated